front/src/locales/translations/ua.json

```diff
@@ -39,5 +39,53 @@
     "en": "Англійська",
     "ru": "Pосійський",
     "ua": "Українська"
+  },
+  "userSettings": {
+    "title": "Загальні налаштування",
+    "lastName": {
+      "label": "Прізвище",
+      "hint": "Введіть прізвище"
+    },
+    "firstName": {
+      "label": "Ім'я",
+      "hint": "Введіть ім'я"
+    },
+    "email": {
+      "label": "Скринька",
+      "hint": "Приклад: bereznievatiy@gmail.com",
+      "errors": {
+        "regexNotMatch": "Введіть дійсний email",
+        "alreadyTaken": "Цей email вже зайнятий"
+      }
+    },
+    "gender": {
+      "label": "Стать",
+      "options": {
+        "NotSpecified": "Стать не вказана",
+        "Male": "Чоловіча",
+        "Female": "Жіноча",
+        "Other": "Інша"
+      }
+    },
+    "language": {
+      "label": "Мова",
+      "options": {
+        "English": "Англійська",
+        "Russian": "Російська",
+        "Ukrainian": "Українська",
+        "German": "Німецька"
+      }
+    },
+    "country": {
+      "Label": "Країна",
```

Comment:
it is way to promote copypast in future. Is is vocabulary. Keys can be logical, but not ui-based. No label, options key